### PR TITLE
Add Melissa Van Bussel to collection of blogs

### PR DESCRIPTION
**Pull request by melissavanbussel**

### CODE REVIEW

**Comment by melissavanbussel** [comment]
Tagging @drmowinckels for review :)

File: blogs/youtube.com.c.ggnot2.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "title": "Melissa Van Bussel (ggnot2)'s YouTube channel about R",
+  "type": "blog",
```
Comment (cosimameyer) [inline blogs/youtube.com.c.ggnot2.json:3]: Can we change this one to `YouTube` (instead of `blog`)?

**Review by cosimameyer — CHANGES_REQUESTED** [review]
Just one small request, everything else looks good 🙌

Thanks so much for contributing ☺️

File: blogs/youtube.com.c.ggnot2.json (hunk 1 above)
Comment (melissavanbussel) [reply blogs/youtube.com.c.ggnot2.json]: Oops, didn't realize that was an option! Updated :) Thank you!

**Review by cosimameyer — APPROVED** [review]

**Comment by cosimameyer** [comment]
Amazing, thanks! 🥳

**Pull request merged**